src/views/EditSidebar.vue

```diff
@@ -84,6 +84,9 @@
 				:href="downloadURL">
 				{{ $t('calendar', 'Download') }}
 			</ActionLink>
+			<ActionButton v-if="!isNew" icon="icon-category-office" @click="duplicateEvent()">
```

Comment:
We moved to material design icons in the meantime, so this icon needs to be adjusted. I would propose
https://materialdesignicons.com/icon/content-duplicate